include/mata/util.hh

```diff
@@ -580,7 +580,7 @@ void filter(Vector & vec, F && is_staying) {
 
         // remove duplicates
         auto it = std::unique(vec.begin(), vec.end());
-        vec.reserve(it - vec.begin());
+        vec.reserve(static_cast<unsigned long>(it - vec.begin()));
```

Comment:
I believe this should be `size_t` (which on our architectures translates to `unsigned long`). That would better explain the purpose of this cast. I will change it myself. No need to bother you with this issue.

Reply:
Done.